core/test/ModelArray_test.cpp

```diff
@@ -255,6 +255,36 @@ TEST_CASE("Location from index")
     REQUIRE(loc[1] == y);
     REQUIRE(loc[2] == z);
 }
+
+// Test the zIndexAndLayer function
```

Comment:
I think you should expand the description a little bit. At least something similar to issue #296

Reply:
Copied from the #296 description (with minor editing).